demos/sprint0/20-dev-malcolm-server.md

```diff
@@ -0,0 +1,35 @@
+# 20 - Make a Malcolm socket simulator
+
+## Demo
+
+1. Start the dev server
+    ```
+    npm run server:dev
+    ```
+1. Start the interactive socket demo server
```

Comment:
client?

Reply:
no, there is a demo js file in the sprint0 folder that allows me to query the server from the command line - this is simply to prove that I've set up the test malcolm server correctly without having to spin up MalcolmJS

Reply:
ah did you mean this should say "socket demo client"? Yes, probably. It's a little confusing as I run the demo app as a node server but read from the command line.

Reply:
I'm planning to fix the wording in my next pull request with other demos.